horizon_duration: 1.5
n_nodes: 30

define:
    - &low_stiff 200.
    - &low_damp 10.
    - &high_stiff 800.
    - &high_damp 10.

stiffness:
    left_hip_pitch_joint: *high_stiff
    left_hip_roll_joint: *high_stiff
    left_hip_yaw_joint: *high_stiff
    left_knee_joint: *high_stiff
    left_ankle_pitch_joint: 500
    left_ankle_roll_joint: 500
    right_hip_pitch_joint: *high_stiff
    right_hip_roll_joint: *high_stiff
    right_hip_yaw_joint: *high_stiff
    right_knee_joint: *high_stiff
    right_ankle_pitch_joint: 500
    right_ankle_roll_joint: 500
    waist_yaw_joint: *low_stiff
    waist_roll_joint: *low_stiff
    waist_pitch_joint: *low_stiff
    left_shoulder_pitch_joint: *low_stiff
    left_shoulder_roll_joint: *low_stiff
    left_shoulder_yaw_joint: *low_stiff
    left_elbow_joint: *low_stiff
    left_wrist_roll_joint: *low_stiff
    left_wrist_pitch_joint: *low_stiff
    left_wrist_yaw_joint: *low_stiff
    right_shoulder_pitch_joint: *low_stiff
    right_shoulder_roll_joint: *low_stiff
    right_shoulder_yaw_joint: *low_stiff
    right_elbow_joint: *low_stiff
    right_wrist_roll_joint: *low_stiff
#
#
damping:
    left_hip_pitch_joint: *high_damp
    left_hip_roll_joint: *high_damp
    left_hip_yaw_joint: *high_damp
    left_knee_joint: *high_damp
    left_ankle_pitch_joint: *high_damp
    left_ankle_roll_joint: *high_damp
    right_hip_pitch_joint: *high_damp
    right_hip_roll_joint: *high_damp
    right_hip_yaw_joint: *high_damp
    right_knee_joint: *high_damp
    right_ankle_pitch_joint: *high_damp
    right_ankle_roll_joint: *high_damp
    waist_yaw_joint: *high_damp
    waist_roll_joint: *high_damp
    waist_pitch_joint: *high_damp
    left_shoulder_pitch_joint: *high_damp
    left_shoulder_roll_joint: *high_damp
    left_shoulder_yaw_joint: *high_damp
    left_elbow_joint: *high_damp
    left_wrist_roll_joint: *high_damp
    left_wrist_pitch_joint: *high_damp
    left_wrist_yaw_joint: *high_damp
    right_shoulder_pitch_joint: *high_damp
    right_shoulder_roll_joint: *high_damp
    right_shoulder_yaw_joint: *high_damp
    right_elbow_joint: *high_damp
    right_wrist_roll_joint: *high_damp

